### PR TITLE
Add Getting Started doc retrieval, move docs data up to top level

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
Add some UI fixes for loading, nav bar collapse, quick start launch buttons

### CODE REVIEW

**Review by cdcabrera — APPROVED** [review]
looks ok

at some point may want to rethink the use of redux vs context since it appears we're going full hooks. some of the sharing, or lack of by using services directly in components may come back around but for now good

File: frontend/src/app/App.scss [hunk 1]
```diff
@@ -20,10 +25,15 @@ body {
...
+  // Bootstrap Overrides (Bootstrap pulled in by @cloudmosaic/quickstarts)
+  code {
+    color: initial;
+    background-color: initial;
+  }
 }
 
 // PF Overrides
-@media (min-width: 800px) {
+@media (min-width: 768px) {
```
[the remark this answers is not in the document]
Comment (jeff-phillips-18) [reply frontend/src/app/App.scss:36]: Seems we have access, but using it in the media query doesn't work for whatever reason.

**Comment by jeff-phillips-18** [comment]
I've left Redux in for now since I'm not fully aware of what else might be coming.

File: frontend/src/app/App.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (cdcabrera) [reply frontend/src/app/App.scss]: hmmm, i seem to remember trying this being an issue on another app i'll take a look see what we did there

**Review by cdcabrera — APPROVED** [review]
lgtm

File: frontend/src/app/App.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (cdcabrera) [reply frontend/src/app/App.scss]: looks like we stuck to using the scss value instead in those other scenarios

**Pull request merged**